app/helpers/concepts_helper.rb

```diff
@@ -132,15 +132,14 @@ def render_concepts_by_dates(auto_click: false)
     tmp = {}
     tmp[first_year] = first_month_concepts
     @concepts_year_month = tmp.merge(@concepts_year_month)
-
+    first_concept_id = @concepts_year_month.first.last.first.last.first.id if auto_click
```

Comment:
this obviously not correct.

Choose the selected concept in the controller level, only if the page is equal to 1, and add it as an argument.

Reply:
auto_click means it's the first page, hhh and it's working why obviously not correct

Reply:
`@concepts_year_month` came from the controller, using `@concepts`. 

so  `first_concept = @concepts.first.id if params[:page].eql?(1)` 

The `auto_click` goal was to say if you want to auto-click using js on the selected concept. SO that when the Date view is displayed it auto_clicks on the selected component to update the concepts details view.